### PR TITLE
CLI test coverage rised to 100%, Registry integration tests added AEA-629

### PR DESCRIPTION
**Pull request by panasevychol**
## Proposed changes

CLI test coverage rised to 100%, Registry integration tests added.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/cli/interact.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def _run_interaction_channel():
 
     try:
         multiplexer.connect()
-        while True:
+        while True:  # pragma: no cover
```
Comment (solarw) [inline aea/cli/interact.py:77]: this code looks like primary logic
not sure should skip coverage for this while cycle

Comment (panasevychol) [reply aea/cli/interact.py:77]: @solarw how do you propose to test this infinite loop with ctrl+c break?

Comment (DavidMinarsch) [reply aea/cli/interact.py:77]: You can extract the logic in the loop in a helper function; then just test the function

File: tests/test_cli/test_add/test_connection.py [hunk 2]
```diff
@@ -468,3 +473,18 @@ def teardown_class(cls):
             shutil.rmtree(cls.t)
         except (OSError, IOError):
             pass
+
+
+@pytest.mark.integration
```
Comment (DavidMinarsch) [inline tests/test_cli/test_add/test_connection.py:478]: @panasevychol for now mark also `unstable` so it does not pollute our CI

File: tests/test_cli/test_add/test_contract.py [hunk 3]
```diff
@@ -50,3 +54,18 @@ def test_add_contract_positive(self, *mocks):
             standalone_mode=False,
         )
         self.assertEqual(result.exit_code, 0)
+
+
+@pytest.mark.integration
```
Comment (DavidMinarsch) [inline tests/test_cli/test_add/test_contract.py:59]: see above re unstable

File: tests/test_cli/test_add/test_protocol.py [hunk 4]
```diff
@@ -465,3 +469,18 @@ def teardown_class(cls):
             shutil.rmtree(cls.t)
         except (OSError, IOError):
             pass
+
+
+@pytest.mark.integration
+class TestAddProtocolFromRemoteRegistry(AEATestCaseEmpty):
```
Comment (DavidMinarsch) [inline tests/test_cli/test_add/test_protocol.py:475]: see above

File: tests/test_cli/test_add/test_skill.py [hunk 5]
```diff
@@ -491,3 +494,18 @@ def test_add_skill_with_contracts_positive(self):
         contracts_folders = os.listdir(contracts_path)
         contract_dependency_name = "erc1155"
         assert contract_dependency_name in contracts_folders
+
+
+@pytest.mark.integration
+class TestAddSkillFromRemoteRegistry(AEATestCaseEmpty):
```
Comment (DavidMinarsch) [inline tests/test_cli/test_add/test_skill.py:500]: see above

File: tests/test_cli/test_fetch.py [hunk 6]
```diff
@@ -137,3 +141,14 @@ def test__is_version_correct_negative(self):
         public_id_mock.version = "incorrect"
         result = _is_version_correct(ctx_mock, public_id_mock)
         self.assertFalse(result)
+
+
+@pytest.mark.integration
+class TestFetchFromRemoteRegistry(AEATestCaseMany):
```
Comment (DavidMinarsch) [inline tests/test_cli/test_fetch.py:147]: see above

**Pull request merged**